default/stringtables/en.txt

```diff
@@ -2410,6 +2416,12 @@ Mature
 GSETUP_ANCIENT
 Ancient
 
+GSETUP_STARTING_ERA_PREWARP
+Pre-Warp
+
+GSETUP_STARTING_ERA_CLASSICAL
+Classical
```

Comment:
Perhaps not the best name for civs able to travel FTL...

Reply:
Changed in commit 1bdbfc42.